src/command/Command.cpp

```diff
@@ -255,13 +255,14 @@ void Command::actionTopic(Client& client) {
         if (topic[0] == ':') {
             it->second.setTopic(param_.back());
```

Comment:
Topic is being set straight to the parameter.. it should strip the colon beforehand.
Also if the parameter is only a colon, it should set the topic to an empty string.